### PR TITLE
schema and pkh refactor

### PR DESCRIPTION
**Pull request by decentralgabe**
- update credential manifest schema from https://github.com/decentralized-identity/credential-manifest/pull/122
- minor pkh refactor
- moved out non-schemas from schema dir

### CODE REVIEW

File: did/pkh.go [hunk 1]
```diff
@@ -29,24 +33,29 @@ const (
...
+	EthereumNetworkPrefix = "eip155:1"
+	PolygonNetworkPrefix  = "eip155:137"
 
-// The following context should be manually inserted into each DID Document. This will likely change
-// over time as new verification methods are supported, and general-purpose methods are specified.
-var knownDIDPKHContext, _ = schema.GetKnownSchema(pkhContextFilename)
+	EcdsaSecp256k1RecoveryMethod2020 = "EcdsaSecp256k1RecoveryMethod2020"
+)
+
+// GetDIDPKHContext returns a context  which should be manually inserted into each did:pkh document. This will likely
```
Comment (nitro-neal) [inline did/pkh.go:44]: two spaces between context and which

File: did/pkh.go [hunk 2]
```diff
@@ -82,14 +91,58 @@ func (d DIDPKH) Method() Method {
...
+		if strings.Contains(did, prefix+":") {
+			return GetDIDPKHNetworkForPrefix(prefix)
 		}
 	}
-	return nil, util.LoggingNewError("network not supported")
+	return "", fmt.Errorf("could not find network for did:pkh DID: %s", did)
+}
+
+// GetVerificationTypeForNetwork returns the verification key type for a given network
+func GetVerificationTypeForNetwork(n Network) (string, error) {
```
Comment (nitro-neal) [inline did/pkh.go:132]: Nice, we don't have to have a mapping anymore, there could be a case where we add a new pkh but forget to add it here in the GetVerificationTypeForNetwork function, thats why I opt to have one source of truth mapping. But this works good too

**Review by nitro-neal — APPROVED** [review]
Looks good!

File: dwn/model.go [hunk 3]
```diff
@@ -53,27 +47,28 @@ func (msg *DWNMessage) IsValid() error {
...
-		return errors.Wrap(err, "could not get dwn message schema")
-	}
-	if err = schema.IsJSONValidAgainstSchema(string(jsonBytes), s); err != nil {
-		logrus.WithError(err).Errorf("dwn message not valid against schema")
-		return err
-	}
-	return nil
+// IsValidDWNMessage validates a given dwn message object against its known JSON schema
+func IsValidDWNMessage(msg DWNMessage) error {
+	// TODO: add a schema when there is one
```
[the remark this answers is not in the document]
Comment (decentralgabe) [reply dwn/model.go:60]: changing this to link to https://github.com/TBD54566975/ssi-sdk/issues/62 to re-add schemas

File: did/pkh.go (hunk 2 above)
Comment (decentralgabe) [reply did/pkh.go:132]: good point, let me see how to define this once

**Pull request merged**